server/lib/device/device.saveState.js

```diff
@@ -13,6 +14,10 @@ const { EVENTS, WEBSOCKET_MESSAGE_TYPES } = require('../../utils/constants');
  * }, 12);
  */
 async function saveState(deviceFeature, newValue) {
+  if (Number.isNaN(newValue)) {
```

Comment:
I hope that most integration are sending state as Number and not string

Because before, you could send: "12" and it would work, now "12" rejects (I agree that it's a good thing)

By checking the code: 

- [x] MQTT
- [x] Zigbee2mqtt
- [x] Bluetooth 
- [x] Philips Hue
- [ ] Tasmota? (Not sure about this one)
- [x] Xiaomi
- [x] ZWave

Reply:
`isNaN` function handles numbers as string.
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/isNaN

According to code, Tasmota implemented features only use numbers.

Reply:
Indeed, Number.isNaN only return true for "NaN" values, my bad!